monkey/infection_monkey/network_scanning/mssql_fingerprinter.py

```diff
@@ -77,27 +80,51 @@ def _query_mssql_for_instance_data(host: str) -> Optional[bytes]:
         sock.close()
 
 
-def _get_services_from_server_data(data: bytes) -> Dict[str, Any]:
-    services = {MSSQL_SERVICE: {}}
-    services[MSSQL_SERVICE]["display_name"] = DISPLAY_NAME
-    services[MSSQL_SERVICE]["port"] = SQL_BROWSER_DEFAULT_PORT
+def _get_instance_info(instance_info: Sequence[str]) -> Dict[str, str]:
+    info = {}
+    it = iter(instance_info)
+    for k in it:
+        info[k] = next(it)
+
+    return info
```

Comment:
This is confusing. This method doesn't get any info, it just creates a dict from sequence. And even then, the dict is d[key] = key, which is questionable at the very least.

Reply:
It is not d[key] = key

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/15820737/220129355-07a7217c-65ff-4634-b56f-3b314c382066.png">